www/components/Layout/Drawer/Drawer.js

```diff
@@ -25,26 +24,19 @@ const Drawer = () => {
           startIcon={<CalendarMonthIcon />}
           variant="text"
           size="large"
-          className={styles.icon_button + " " + styles.blue}
-          onClick={() => router.push(`/user/${user._id}/home`)}
-        >
-          Upcoming
-        </Button>
-        <Button
-          startIcon={<EventAvailableIcon />}
-          variant="text"
-          size="large"
           className={styles.icon_button + " " + styles.green}
+          onClick={() => router.push(`/user/${user._id}/home`)}
         >
-          Completed
+          Meetings
         </Button>
         <Button
-          startIcon={<DescriptionIcon />}
+          startIcon={<TaskAltIcon />}
           variant="text"
           size="large"
-          className={styles.icon_button + " " + styles.purple}
+          className={styles.icon_button + " " + styles.blue}
+          onClick={() => router.push(`/user/${user._id}/tasks`)}
         >
-          Drafts
+          Action items
```

Comment:
No way, the drawer actually works now!?!?!!?!

Reply:
It actually has a function!